lib/main.coffee

```diff
@@ -1,114 +1,80 @@
-{CompositeDisposable} = require 'atom'
+CSON = require 'season'
 path = require 'path'
-promiseHelper = require './promise-helper'
+
+grammarRE = /\/language-[^\/]+\/grammars\/[^\/]+\.(?:c|j)son/
+
+getPackageByPath = (path) ->
+  for name, pack of atom.packages.loadedPackages
+    return pack if pack.path is path
+
+readCsonFile = (filePath) ->
```

Comment:
Restore this method in this original place `promise-helper.coffee`.

Reply:
If we add integration tests, there's no need for `readCsonFile` to have its own unit test, and thus no need for its own file.